vendor/k8s.io/kubernetes/pkg/kubeapiserver/authorizer/modes/patch.go

```diff
@@ -0,0 +1,8 @@
+package modes
+
+var ModeScope = "Scope"
+var ModeSystemMasters = "SystemMasters"
```

Comment:
why not also BrowserSafe?

Reply:
> why not also BrowserSafe?

Because that change the resources, but the point of SystemMasters is to approve everything, no matter the resource.

Reply:
as another authz-mode? Which resource?

Reply:
> as another authz-mode? Which resource?

it wraps the attributes with an adapter than can conditionally return a different verb and resource for /proxy to require a second rule to allow it so that always comes from an authenticated request.